pacemaker_changenotifier/lib/models/workout_list_model.dart

```diff
@@ -40,7 +40,7 @@ class WorkoutListModel extends ChangeNotifier {
       _isLoading = false;
       setTitle();
       notifyListeners();
-    }).catchError((err) {
+    }).catchError((dynamic err) {
```

Comment:
How about printing the error to the console? Otherwise, you won't know what the error is